integration_tests/dbt_project.yml

```diff
@@ -61,12 +61,33 @@ vars:
 seeds:
   quickbooks_source_integration_tests:
     +quote_columns: "{{ true if target.type == 'redshift' else false }}"
-  
+    +column_types:
```

Comment:
We will need to apply the same updates in the `dbt_quickbooks` package too.

Reply:
done